lib/signal_tower/room.ex

```diff
@@ -128,14 +132,35 @@ defmodule SignalTower.Room do
     end)
   end
 
-  defp send_joined_room(pid, peer_id, members) do
-    response_for_joined_peer = %{
-      event: "joined_room",
-      own_id: peer_id,
-      peers: members |> Map.values()
-    }
+  defp send_joined_room(pid, own_id, members, last_turn_timestamp) do
+    now = System.os_time(:second)
+
+    {turn_response, next_turn_timestamp} =
+      if System.get_env("SIGNALTOWER_TURN_SECRET") && last_turn_timestamp < now do
+        next_timestamp = now + 3 * 60 * 60
```

Comment:
I used that before, but since it's only possible to get the system time as unix time, so I would need to always turn that into elixir time and also carry the turn_timestamp around as elixir time only to change it again to unix time for export - so since both ends are unix time, I'm not sure if it makes sense to make it into elixir time in between only to use the add function here?

Reply:
I could just imagine that this could be a source of failure since the timestamp has no type based unit. Also the variable name does not provide any information about its unit.